### PR TITLE
Added boilerplate and created a connection to the pokemon database

### PR DESCRIPTION
**Pull request by EleanorRoberts**

### CODE REVIEW

File: collectionchallenge.php [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+<!--	<link rel="stylesheet" href="normalize.css" />-->
+<!--	<link rel="stylesheet" href="xxx.css" />-->
+<!--	<meta name="viewport" content="width=device-width, initial-scale=1.0" />-->
+
+</head>
+<body>
+
+</body>
+</html>
```
Comment (Roomsycakes01) [inline collectionchallenge.php:41]: extra line

File: collectionchallenge.php [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+?>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8" />
+    <title>Pokemon Collection</title>
+
+<!--	<link rel="stylesheet" href="normalize.css" />-->
+<!--	<link rel="stylesheet" href="xxx.css" />-->
+<!--	<meta name="viewport" content="width=device-width, initial-scale=1.0" />-->
```
Comment (Roomsycakes01) [inline collectionchallenge.php]: may as well uncomment this

File: collectionchallenge.php [hunk 3]
```diff
@@ -0,0 +1,25 @@
+<?php
```
Comment (Roomsycakes01) [inline collectionchallenge.php:1]: maybe keep the php and html separate by using a require once?

File: collectionchallenge.php [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+$db = new PDO('mysql:host=db; dbname=pokemon', 'root', 'password' );
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+$query = $db->prepare("SELECT `stats`.`name`, `stats`.`type1`, `stats`.`type2`, `stats`.`hp`, `stats`.`attack`, `stats`.`defense`, `stats`.`spAttack`, `stats`.`spDefense`, `stats`.`speed` FROM `stats`;");
+$query->execute();
+$result = $query->fetchAll();
+
+var_dump($result);
```
Comment (Roomsycakes01) [inline collectionchallenge.php]: not needed in final so maybe don't merge this

File: collectionchallenge.php [hunk 5]
```diff
@@ -0,0 +1,25 @@
+<?php
+
+$db = new PDO('mysql:host=db; dbname=pokemon', 'root', 'password' );
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+$query = $db->prepare("SELECT `stats`.`name`, `stats`.`type1`, `stats`.`type2`, `stats`.`hp`, `stats`.`attack`, `stats`.`defense`, `stats`.`spAttack`, `stats`.`spDefense`, `stats`.`speed` FROM `stats`;");
```
Comment (Roomsycakes01) [inline collectionchallenge.php]: maybe make a more specific variable name for your query as I'm guessing you'll want to just pull items by name and have php pull the rest which will require multiple queries or different fetches.

File: collectionchallenge.php [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+var_dump($result);
+?>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8" />
+    <title>Pokemon Collection</title>
+
+<!--	<link rel="stylesheet" href="normalize.css" />-->
+<!--	<link rel="stylesheet" href="xxx.css" />-->
```
Comment (Roomsycakes01) [inline collectionchallenge.php:12]: Raunchy ;)

File: collectionchallenge.php [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+
+var_dump($result);
+?>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8" />
+    <title>Pokemon Collection</title>
+
+<!--	<link rel="stylesheet" href="normalize.css" />-->
```
Comment (Roomsycakes01) [inline collectionchallenge.php:11]: May as well get this implemented by pasting in normalize before pulling

**Review by Roomsycakes01 — COMMENTED** [review]
looks good! The changes I suggested were mainly pedantic and for ease of use in future tasks :)

File: collectionchallenge.php (hunk 1 above)
Comment (EleanorRoberts) [reply collectionchallenge.php:41]: For content

File: collectionchallenge.php (hunk 2 above)
Comment (EleanorRoberts) [reply collectionchallenge.php]: Currently has invalid href

File: collectionchallenge.php (hunk 4 above)
Comment (EleanorRoberts) [reply collectionchallenge.php]: Removed from file, there purely to show the database was correctly connected

File: collectionchallenge.php (hunk 3 above)
Comment (EleanorRoberts) [reply collectionchallenge.php:1]: Added

File: collectionchallenge.php (hunk 5 above)
Comment (EleanorRoberts) [reply collectionchallenge.php]: Currently pulling all of my data with one query

Comment (EleanorRoberts) [reply collectionchallenge.php]: Updated anyway

**Review by jesshamby — APPROVED** [review]
All looks great to me :)

**Pull request merged**